tools/ci/travis/travis-ci.sh

```diff
@@ -165,12 +165,15 @@ CheckManual () {
       cat<<EOF
 --------------------------------------------------------------------------
 This test checks that all standard library modules are referenced by the
-standard library chapter of the manual.
+standard library chapter of the manual and that there are no name collision
+between latex source file and module documentation on a case-insensitive
+file system.
```

Comment:
Maybe you could be more abstract here ("Performs sanity checks on the manual" or something) to avoid having to extend it again (redundantly with the Makefile) if we add new checks in the future.

Reply:
I have reorganised the documentation a bit in order to keep this text minimalist and increase the information content of the related Makefile.